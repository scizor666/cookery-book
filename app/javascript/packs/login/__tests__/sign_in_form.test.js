import React from 'react';
import renderer from 'react-test-renderer';
import SignInForm from '../sign_in_form';


test.skip('display when loading', () => {
    const wrapper = renderer.create(<SignInForm csrfToken="someTokenValueHere"/>);
    expect(wrapper.toJSON()).toMatchSnapshot();
});