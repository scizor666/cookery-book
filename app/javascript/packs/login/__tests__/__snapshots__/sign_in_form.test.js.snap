// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`display when loading 1`] = `
<div
  className="row"
>
  <div
    className="col-md-3 col-md-offset-3 container"
  >
    <h1
      className="text-center"
    >
      Log in
    </h1>
    <form
      acceptCharset="UTF-8"
      action="/login"
      method="post"
    >
      <input
        name="utf8"
        type="hidden"
        value="âœ“"
      />
      <input
        name="authenticity_token"
        type="hidden"
        value="someTokenValueHere"
      />
      <div
        className="form-group"
      >
        <label
          htmlFor="session_email"
        >
          Email
        </label>
        <input
          className="form-control"
          id="session_email"
          name="session[email]"
          type="email"
        />
        <label
          htmlFor="session_password"
        >
          Password
        </label>
        <input
          className="form-control"
          id="session_password"
          name="session[password]"
          type="password"
        />
        <label
          className="checkbox inline"
          htmlFor="session_remember_me"
        >
          <input
            name="session[remember_me]"
            type="hidden"
            value="0"
          />
          <input
            id="session_remember_me"
            name="session[remember_me]"
            type="checkbox"
            value="1"
          />
          <span>
            Remember me on this computer
          </span>
        </label>
      </div>
      <div
        className="form-group col-md-12 text-center"
      >
        <input
          className="btn btn-primary"
          data-disable-with="Log in"
          name="commit"
          type="submit"
          value="Log in"
        />
      </div>
      New user? 
      <a
        href="/users/new"
      >
        Sign up now!
      </a>
    </form>
  </div>
</div>
`;
